test/create-user.test.ts

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+import bcrypt from 'bcrypt';
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { AppDataSource } from '../src/data-source.js';
+import { User } from '../src/data/db/index.js';
```

Comment:
esses imports estao esquisitos 👀 
acho que vc vai ter que colocar toda a pasta test dentro da pasta src. Geralmente a flag `rootDir` impede coloca arquivos ts fora da src, da uma olhada se for isso q ta dando erro